src/main/scala/vsys/blockchain/contract/ContractVStableSwap.scala

```diff
@@ -122,29 +125,32 @@ object ContractVStableSwap {
   val withdrawId: Short = 2
   val withdrawPara: Seq[String] = Seq("withdrawer", "amount", "tokenId") ++
                                   Seq("baseTokenId", "targetTokenId", "isValidTokenId",
-                                      "isBaseToken", "baseTokenIfBlock", "isTargetToken", "targetTokenIfBlock")
+                                      "isBaseToken", "valueFalse", "baseTokenIfBlock", "isTargetToken", "targetTokenIfBlock")
```

Comment:
it seems that both deposit and withdraw triggers add this TokenId check?

Reply:
you are right, this is inconsistent, I will remove it from the withdraw trigger.